services/osparc-gateway-server/Dockerfile

```diff
@@ -89,7 +88,6 @@ WORKDIR /build
 COPY --chown=scu:scu services/osparc-gateway-server/requirements/_base.txt .
 
 # install base 3rd party dependencies (NOTE: this speeds up devel mode)
-ENV MAKEFLAGS="-j 12"
```

Comment:
So this works in parallel without that flag?

Reply:
hmm, so first I did not see any difference when using it. Then since I migrated to python 3.9, the packages do not need to be built since they are available on piwheels.org